documentation/List-of-Action-plugins.md

```diff
@@ -461,7 +460,7 @@ See [Grant Permissions](#grant-permissions) above regarding acting on user-only
 
 ### Generate Review Assignments
 
-Generates records in the `review_assignment` table -- i.e. which users (reviewers) are allowed to do a review for the current stage/level (and for which Sections). The records are set with `status` "Available" or "Assigned" and flags for `isSelfAssignable` and `isLocked` to help define when is allowed self-assignment.
+Generates records in the `review_assignment` table -- i.e. which users (reviewers) are allowed to do a review for the current stage/level (and for which Sections). The records are set with `status` "Available" or "Assigned". With the properties to specify the type of assignament: `isSelfAssignable` if it should show for self-assignment when not assigned by another user. The `isLocked` property helps defining that the review can start but not be submitted - after the application has been back to Applicant for ammendments.
```

Comment:
Maybe bullet-point these properties to make easier to read?

Also:
> `isSelfAssignable` if it should show for self-assignment when not assigned by another user.

Isn't "self-assignable" a template configuration? So maybe should say "determines whether this assignment is self-assignable as determined by the template configuration for this stage/level" ?

Reply:
Typo on `assignament` => "assignment" ;)

Reply:
Thanks. Done